src/features/view-nft/NFTDetailsCard/ui/Options/Options.module.scss

```diff
@@ -1,22 +1,24 @@
 @import '../../../../../../node_modules/@zero-tech/zui/styles/_borders.scss';
 @import '../../../../../../node_modules/@zero-tech/zui/styles/_colors.scss';
 
-.Container {
+.Options {
 	display: flex;
 	align-items: center;
 	background: $purple-10;
 	border-radius: calc($radius-primary / 2);
 	height: 2.313rem;
+	z-index: 2;
```

Comment:
added z-index to bring tooltip to the front - previously displaying behind nft image